project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -183,4 +183,36 @@ public static Document getDocumentByHash(String hash) {
 
     return docs;
   }
+
+  //Takes in a Document hash and a User email
```

Comment:
Formatting nit: we usually have two spaces after // - what's the rule of your linter?

Reply:
Seems to be one space, ran linter to fix it